src/main/java/org/launchcode/liftoffrecipetracker/models/Beverage.java

```diff
@@ -0,0 +1,39 @@
+package org.launchcode.liftoffrecipetracker.models;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+    @Entity
+    public class Beverage extends AbstractEntity {
+
+        //Class Variables
+        @Size(min = 4, max = 50, message = "Beverage name must be between 4 and 50 characters long")
+        @NotBlank
+        @NotNull
+        private String name;
```

Comment:
This is a great start to a really cool idea--we could definitely expand it to include extra details for vintage, cocktail ingredients, etc. Looking forward to seeing it expand!